src/protagonist/API/Features/Image/Validation/HydraImageValidator.cs

```diff
@@ -28,29 +28,26 @@ public HydraImageValidator(IOptions<ApiSettings> apiSettings)
             RuleFor(a => a.MediaType).NotEmpty().WithMessage("Media type must be specified");
         });
         
-        When(a => !a.WcDeliveryChannels.IsNullOrEmpty(), DeliveryChannelDependantValidation)
-            .Otherwise(() =>
-            {
-                RuleFor(a => a.Width).Empty().WithMessage("Should not include width");
-                RuleFor(a => a.Height).Empty().WithMessage("Should not include height");
-                RuleFor(a => a.Duration).Empty().WithMessage("Should not include duration");
-            });
-        
         When(a => !a.DeliveryChannels.IsNullOrEmpty(), ImageDeliveryChannelDependantValidation);
         
+        // Legacy policy fields
+        RuleFor(a => a.ImageOptimisationPolicy)
+            .Null()
+            .When(_ => !apiSettings.Value.EmulateOldDeliveryChannelProperties)

```

Comment:
Will these be expanded in next ticket? (to capture that there are only some policies that we'll accept)